src/http/sr-modules-operators-keys/sr-modules-operators-keys.controller.ts

```diff
@@ -79,4 +81,28 @@ export class SRModulesOperatorsKeysController {
       { isolationLevel: IsolationLevel.REPEATABLE_READ },
     );
   }
+
+  @Version('2')
+  @ApiOperation({ summary: 'Comprehensive stream for staking router modules, operators and their keys' })
+  @ApiResponse({
+    status: 200,
+    description: 'Stream of all SR modules, operators and keys',
+    type: SRModulesOperatorsKeysStreamResponse,
+  })
+  @ApiResponse({
+    status: 425,
+    description: 'Meta has not exist yet, maybe data was not written in db yet',
+    type: TooEarlyResponse,
+  })
+  @Get('operators/keys')
+  async getModulesOperatorsKeysStream(@Res() reply: FastifyReply) {
+    const jsonStream = JSONStream.stringify();
+
+    reply.type('application/json').send(jsonStream);
+
+    await this.entityManager.transactional(
+      () => pipeline([streamify(this.srModulesOperatorsKeys.getModulesOperatorsKeysGenerator()), jsonStream]),
```

Comment:
What will happen if error happen during stream processing ? is it possible that we will send successfully part of data?

Reply:
will be fixed in another task